src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
```

Comment:
```suggestion
  const calculator = {
    result: 0,
```
Let's use more meaningful names)
![image](https://user-images.githubusercontent.com/96409563/209784082-ff204e81-aa85-4dc0-b425-a9f1f662dba8.png)